bcda/models/postgres/postgrestest/postgrestest.go

```diff
@@ -0,0 +1,364 @@
+// Package postgrestest provides CRUD utilities for the postgres database.
+// These utilities allow the caller to modify the database in ways that we
+// wouldn't want to permit in the main code path.
+// To protect against usage in non-test code, all methods should accept
+// a *testing.T struct.
```

Comment:
Love this, if I didn't already say that somewhere else.  It's been a long review :)